File: Modul-02/C/PRAK204-2310817210009-Muhammad-Ibnu-Sina.c
#include <stdio.h>

int main() {
    float phi = 3.142857;
    float r, t;

    printf("Input: \n");
    scanf("%f %f", &r, &t);

    float volume = phi * r * r * t;
    float luas = 2 * phi * r * (r + t);
    float keliling = 2 * phi * r;

    printf("Output: \n");
    printf("Volume : %.2f\n", volume);
    printf("Luas : %.2f\n", luas);
    printf("Keliling : %.2f\n", keliling);

    return 0;
}

File: Modul-02/python/PRAK204-2310817210009-Muhammad-Ibnu-Sina.py
import math


def main():
    phi = 3.142857

    print("Input: ")
    r, t = map(float, input().split())

    volume = phi * r * r * t
    luas = 2 * phi * r * (r + t)
    keliling = 2 * phi * r

    print("Output: ")
    print(f"Volume : {volume:.2f}")
    print(f"Luas : {luas:.2f}")
    print(f"Keliling : {keliling:.2f}")


if __name__ == "__main__":
    main()
